Gemfile

```diff
@@ -2,5 +2,6 @@ source 'https://rubygems.org'
 
 ruby "2.5.1"
 
-gem "sinatra"
+gem "sinatra-base"
```

Comment:
Why did you change it to `sinatra-base`?

Reply:
bc that was what wasn't working but I don't really *need* the full sinatra gem I believe

Reply:
Whilst that might be true, I would leave in the whole gem for now :) Whilst definitely a good idea to not bring in more of a dependency than you need, I don't think it's much of a learning prio for now 🙈